src/NexusMods.App.UI/Pages/LocalFileDataProvider.cs

```diff
@@ -23,6 +24,8 @@ public LocalFileDataProvider(IServiceProvider serviceProvider)
         _connection = serviceProvider.GetRequiredService<IConnection>();
     }
 
+    public LibraryFile.ReadOnly[] GetAllFiles(GameId gameId, IDb? db = null) => [];
```

Comment:
Add a comment explaining why this is empty (we don't have game information for local files)
```suggestion
    // TODO: update once we have game information on Local Files
    public LibraryFile.ReadOnly[] GetAllFiles(GameId gameId, IDb? db = null) => [];
```